pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[project]
+name = "hic_qc"
+description = "Quality assessment for Hi-C libraries"
+dynamic = ["version"]
+
+[build-system]
+requires = ["setuptools >= 64.0.0", "setuptools-git-versioning<2"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["."]
+
+[tool.setuptools-git-versioning]
+enabled = true
+template = "v{tag}"
```

Comment:
Presumably this version template works on Github?

Reply:
Yeah, it probably just works then. I'd suggest adding a git tag with that format to main branch HEAD and then installing the conda environment. That will show you the version of the repo that is installed in the details of the conda environment that is installed.

Reply:
I can confirm that it works locally

Reply:
Just to follow up, some additional commits that cleaned up .gitignore had it use a clean version when on the tagged commit.